locale/de_DE/api.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/de_DE/api.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2000-2017 John Willinsky
+  * Copyright (c) 2017 Stefan Weil, UB Mannheim
```

Comment:
Stefan, we cannot accept additional copyrights in the files. You might see that the rest of the translation - mostly done by me - also does not include additional copyright. If you are okay with this, please remove this line. Otherwise, I will include a new translation of the API locale strings when updating the de_DE locale over the next days in preparation for OJS 3.1.